specification/synapse/data-plane/Microsoft.Synapse/preview/2022-12-01-preview/linkConnections.json

```diff
@@ -0,0 +1,1043 @@
+        }
+      }
+    },
+    "/linkconnections/{linkConnectionName}/pause": {
+      "post": {
+        "tags": [
+          "linkConnections"
+        ],
+        "operationId": "LinkConnection_Pause",
+        "description": "Pause a link connection. It may take a few minutes from Pausing to Paused, monitor the status with LinkConnection_GetDetailedStatus.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/LinkConnectionName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "../../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "LinkConnection_Pause": {
+            "$ref": "./examples/LinkConnection_Pause.json"
+          }
+        }
+      }
+    },
+    "/linkconnections/{linkConnectionName}/resume": {
+      "post": {
+        "tags": [
+          "linkConnections"
+        ],
+        "operationId": "LinkConnection_Resume",
+        "description": "Resume a link connection. It may take a few minutes from Resuming to Running, monitor the status with LinkConnection_GetDetailedStatus.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/LinkConnectionName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "../../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "LinkConnection_Resume": {
+            "$ref": "./examples/LinkConnection_Resume.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "LinkConnectionListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List link connection value",
+          "items": {
+            "$ref": "#/definitions/LinkConnectionResource"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "List link connections next link"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "LinkTableListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List link table value",
+          "items": {
+            "$ref": "#/definitions/LinkTableResource"
+          }
+        }
+      }
+    },
+    "LinkConnectionResource": {
```

Comment:
most of the models do not have "description"

they will be required in GA.

Reply:
will review and add missing description in GA PR